spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,147 @@
+require 'rails_helper'
+# rubocop:disable Layout/LineLength
```

Comment:
**Well-written code!**

- [ ] I can see you guys put such efford to this project. However, there are some places that need to be fixed up. For starting, in your spef** folder, I can see `rubocop:disable.. `. Using such thing are not allowed in this project. Please remove all of them. 

> HINT: you can create helper folders and put extra codes into it to reduce the size of these files. 
> If these errors are not able to fix you can reach me out, or you can leave these like that. But in my opinion they are fixable :)

Reply:
@Rustamxon7 
thanks for your good and valuable feedback. 
I apply your feedback and remove it.